### PR TITLE
Crafting, Laser, Bunker and Other Tweaks

### PR DESCRIPTION
**Pull request by panzer1944**
## About The Pull Request
Fixes some busted OP crafting recipes finally, no more 12 R91s from a single blueprint and a few stacks of metal gained in 10 minutes. This PR also reverts my last PRs AER14 and Wattz 2000 nerf for the time being due to popular request. I've decided to attach some NSB changes while I was at it for difficulties sake and using previous iterations of the bunkers and existing bunkers as examples - simply put: Nests. You're not meant to be able to rush in and out, 20 minutes from opening with all the best gear. This PR should hopefully be the nail in the coffin to ensure that no, that isn't possible alone or in a small group in that given timeframe - that's not to say it wont be possible overall, it'll just take longer or more manpower, which is good. 
## Why It's Good For The Game
Keeps people happy with regards to the laser revert, and prevents game-breaking high-end weapon spam. I dont think making six garands with zero weapon parts in use is very balanced in the slightest. This PR fixes that. This PR also greatly encourages teamwork to be utilized when aiming to fully-clear the bunker North Sewer Bunker (s) as a result of the nests and other threats, while simultaneously helping to ensure that going through the entire thing is near enough a requirement if the end loot is to be gained (even if this is mostly in the second versions case), akin to how it is in Raider Bunker where you have to clear the vast majority of it for the end loot. Mining vendor changes have been done to help purge some SS13 stuff from easy access. Scout Carbine change has been done to prevent Rangers being gimped by worse service rifles, as per their request.
## Changelog
- Garand Crafting now requires an Assembly and Receiver
- Secondary Rangemaster Recipe (which exists apparently) changed to mostly match other
- Deagle Crafting now requires super-light alloys
- R91 Crafting now needs an Assembly and Receiver.
- Wattz 2000 Fire-delay reverted to 3.5
- AER14 Fire-delay reverted to 3.5
- Attempts to make some walls in the second version indestructible as to encourage fullclearing further if at all
- Adds nests to both versions for challenge purposes.
- VVs the Second Version abominations in the North-West to actually be somewhat scary while still not having celldamage.
- Removes some static-claws to make way for the nests
- Adds some oil slips into the first North Sewer Bunker
- Removes Survival Medipens, Suit Tokens and miner-kits from the Mining Vendor
- Changes Scout Carbine fire-delay from 5 to 4
- Adds indestructible rusted reinforced wall
- Adds NSB unique Abomhorror
- Changes the integrity value of plastitanium glass to 1000 on Myrios' Request as it was 100 yet harder to make than Plasglass
## Pictures
image
image

### CODE REVIEW

File: _maps/map_files/templates/dungeons/north_sewer_2.dmm [hunk 1]
```diff
@@ -196,7 +196,10 @@
 	id = "hell";
 	name = "Door Control"
 	},
-/turf/closed/wall/r_wall/rust,
+/turf/closed/wall/r_wall/rust{
```
Comment (Superspheres) [inline _maps/map_files/templates/dungeons/north_sewer_2.dmm:199]: Lets refrain from editing vars on objects in StrongDMM and add them as a proper subtype in code.

**Review by Superspheres — CHANGES_REQUESTED** [review]
Remove the varedits and add new subtypes.

Examples:
`/turf/closed/wall/r_wall/rust/explosion_resistant
    explosion_block = 50
    unbreakable = null`
   
`/mob/living/simple_animal/hostile/abomhorror/nsb
    bare_wound_bonus = 20
    faction = list("raider)
    melee_damage_lower = 60
    melee_damage_upper = 70
    obj_damage = 300
    wound_bonus = 20`

**Comment by panzer1944** [comment]
yeee okay. give me a bit and i'll do some editing

File: _maps/map_files/templates/dungeons/north_sewer_2.dmm (hunk 1 above)
Comment (panzer1944) [reply _maps/map_files/templates/dungeons/north_sewer_2.dmm]: Added a new indestructible ver of the reinforced wall instead to fix this.

File: _maps/map_files/templates/dungeons/north_sewer_2.dmm [hunk 2]
```diff
@@ -649,10 +652,11 @@
 /turf/open/floor/plasteel/f13/vault_floor/misc/vaultrust,
 /area/f13/bunker)
 "mn" = (
-/obj/effect/decal/cleanable/dirt,
-/obj/structure/barricade/wooden/strong,
-/turf/open/indestructible/ground/inside/mountain,
-/area/f13/caves)
+/turf/closed/wall/r_wall/rust{
```
[the remark this answers is not in the document]
Comment (panzer1944) [reply _maps/map_files/templates/dungeons/north_sewer_2.dmm]: See prior.

File: _maps/map_files/templates/dungeons/north_sewer_2.dmm [hunk 3]
```diff
@@ -1282,13 +1286,18 @@
 /area/f13/radiation)
 "wv" = (
 /obj/effect/decal/cleanable/dirt,
-/obj/effect/decal/cleanable/blood,
-/obj/item/reagent_containers/food/snacks/meat/slab/human,
-/mob/living/simple_animal/hostile/deathclaw{
-	speed = -2
+/mob/living/simple_animal/hostile/abomhorror{
```
[the remark this answers is not in the document]
Comment (panzer1944) [reply _maps/map_files/templates/dungeons/north_sewer_2.dmm]: Added an NSB unique Abomhorror to fix this

File: _maps/map_files/templates/dungeons/north_sewer_2.dmm [hunk 4]
```diff
@@ -1493,14 +1502,13 @@
...
-/obj/machinery/door/poddoor/shutters{
-	id = "pain";
-	name = "door shutters"
-	},
 /obj/structure/barricade/wooden/planks/pregame,
 /obj/structure/decoration/rag{
 	icon_state = "skin"
 	},
+/obj/machinery/door/poddoor{
+	id = "pain"
```
[the remark this answers is not in the document]
Comment (panzer1944) [reply _maps/map_files/templates/dungeons/north_sewer_2.dmm:1500]: Cool and good

File: _maps/map_files/templates/dungeons/north_sewer_2.dmm [hunk 5]
```diff
@@ -1939,8 +1947,12 @@
 "IL" = (
 /obj/effect/decal/cleanable/dirt,
 /mob/living/simple_animal/hostile/abomhorror{
```
[the remark this answers is not in the document]
Comment (panzer1944) [reply _maps/map_files/templates/dungeons/north_sewer_2.dmm]: Fixed.

File: _maps/map_files/templates/dungeons/north_sewer_2.dmm [hunk 6]
```diff
@@ -1951,7 +1963,10 @@
 	id = "okay";
 	name = "Door Control"
 	},
-/turf/closed/wall/r_wall/rust,
+/turf/closed/wall/r_wall/rust{
```
[the remark this answers is not in the document]
Comment (panzer1944) [reply _maps/map_files/templates/dungeons/north_sewer_2.dmm]: Fixed. See prior.

**Comment by panzer1944** [comment]
Fixes done with these commits. Should be good to go.

**Review by Superspheres — APPROVED** [review]

**Pull request merged**